.stylelintrc

```
@@ -0,0 +1,14 @@
+{
+  "processors": [
+    "stylelint-processor-styled-components"
+  ],
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-styled-components"
+  ],
+  "plugins": [
+    "stylelint-declaration-use-variable"
+  ],
+  "rules": {
+    "sh-waqar/declaration-use-variable": "/color/"  }
```

It'd be better to format it like this
```suggestion
    "sh-waqar/declaration-use-variable": "/color/"
  }
```